dashboard/app.R

```diff
@@ -208,9 +208,18 @@ server <- function(input, output, session) {
   dfStateDeaths <- getData("score_cards_state_deaths.rds")
   dfNationCases = getData("score_cards_nation_cases.rds")
   dfNationDeaths = getData("score_cards_nation_deaths.rds")
-  # TODO handle this kind of error better
-  dfNationDeaths = subset(dfNationDeaths, select = -c(full_location_name))
-  dfNationCases = subset(dfNationCases, select = -c(full_location_name))
+  
+  # Pick out expected columns only
+  covCols = paste0("cov_", COVERAGE_INTERVALS)
+  expectedCols = c("ahead", "geo_value", "forecaster", "forecast_date",
```

Comment:
Worth noting that we don't use all of these columns when processing the data later on (for instance, we don't need "data_source", and we don't currently use "forecast_date"), but might still be a good idea to keep them in, just in case.